packages/boot/test/bootstrapTests/ec-membership-update.test.ts

```diff
@@ -0,0 +1,399 @@
+} from '../../tools/drivers.js';
+
+const wallets = [
+  'agoric1gx9uu7y6c90rqruhesae2t7c2vlw4uyyxlqxrx',
+  'agoric1d4228cvelf8tj65f4h7n2td90sscavln2283h5',
+  'agoric14543m33dr28x7qhwc558hzlj9szwhzwzpcmw6a',
+  'agoric13p9adwk0na5npfq64g22l6xucvqdmu3xqe70wq',
+  'agoric1el6zqs8ggctj5vwyukyk4fh50wcpdpwgugd5l5',
+  'agoric1zayxg4e9vd0es9c9jlpt36qtth255txjp6a8yc',
+];
+
+const highPrioritySenderKey = 'highPrioritySenders';
+
+const offerIds = {
+  propose: { outgoing: 'outgoing_propose' },
+  vote: { outgoing: 'outgoing_vote', incoming: 'incoming_vote' },
+};
+
+const getQuestionId = id => `propose-question-${id}`;
+const getVoteId = id => `vote-${id}`;
+
+export const makeZoeTestContext = async t => {
+  console.time('ZoeTestContext');
+  const swingsetTestKit = await makeSwingsetTestKit(t.log, undefined, {
+    configSpecifier: '@agoric/vm-config/decentral-main-vaults-config.json',
+  });
+
+  const { runUtils, storage } = swingsetTestKit;
+  console.timeLog('DefaultTestContext', 'swingsetTestKit');
+  const { EV } = runUtils;
+
+  await eventLoopIteration();
+
+  // We don't need vaults, but this gets the brand, which is checked somewhere
+  // Wait for ATOM to make it into agoricNames
+  await EV.vat('bootstrap').consumeItem('vaultFactoryKit');
+  console.timeLog('DefaultTestContext', 'vaultFactoryKit');
+
+  // has to be late enough for agoricNames data to have been published
+  const agoricNamesRemotes = makeAgoricNamesRemotesFromFakeStorage(storage);
+  console.timeLog('DefaultTestContext', 'agoricNamesRemotes');
+
+  console.timeEnd('DefaultTestContext');
+  const walletFactoryDriver = await makeWalletFactoryDriver(
+    runUtils,
+    storage,
+    agoricNamesRemotes,
+  );
+
+  const { fromCapData } = makeMarshal(undefined, slotToBoardRemote);
+
+  const getUpdatedDebtLimit = () => {
+    const atomGovernance = unmarshalFromVstorage(
+      storage.data,
+      'published.vaultFactory.managers.manager0.governance',
+      fromCapData,
+      -1,
+    );
+    return atomGovernance.current.DebtLimit.value.value;
+  };
+
+  const governanceDriver = await makeGovernanceDriver(
+    swingsetTestKit,
+    agoricNamesRemotes,
+    walletFactoryDriver,
+    wallets,
+  );
+
+  return {
+    ...swingsetTestKit,
+    storage,
+    getUpdatedDebtLimit,
+    governanceDriver,
+  };
+};
+const test = anyTest as TestFn<Awaited<ReturnType<typeof makeZoeTestContext>>>;
+
+test.before(async t => {
+  t.context = await makeZoeTestContext(t);
+});
+
+test.serial('normal running of committee', async t => {
+  const { advanceTimeBy, storage, getUpdatedDebtLimit, governanceDriver } =
+    t.context;
+
+  const agoricNamesRemotes = makeAgoricNamesRemotesFromFakeStorage(storage);
+  const { VaultFactory, economicCommittee } = agoricNamesRemotes.instance;
+  const { ATOM: collateralBrand, IST: debtBrand } = agoricNamesRemotes.brand;
+
+  const committee = governanceDriver.ecMembers;
+
+  t.log('Accepting all invitations for original committee');
```

Comment:
looks good [in ci](https://github.com/Agoric/agoric-sdk/actions/runs/11156604537/job/31009495789?pr=10164#step:5:6166)